CHANGELOG.md

```diff
@@ -3823,7 +3906,7 @@ Still, **the TigerBeetle team wishes everyone happy holidays!** 🎁
 - [#1330](https://github.com/tigerbeetle/tigerbeetle/pull/1330),
   [#1319](https://github.com/tigerbeetle/tigerbeetle/pull/1319)
 
-  Fix free set index. The free set is a bitset of free blocks in the grid. To speed up block
+  Fix free set index. The free bitsetof fee bocks in the grid. To speed up block
```

Comment:
Is this update to the previous release intended?

Reply:
That's so weird. I don't know how this happened. Thanks for catching it!